spec/kafka_spec.cr

```diff
@@ -1,7 +1,13 @@
 require "./spec_helper"
 
 describe Kafka do
-  # TODO: add tests
-  # can only really do integration tests, as it requires connecting to working kafka setup
-
+  describe Kafka::Consumer do
+    context "#initialize" do
+      it "returns a Kafka::Consumer" do
+        config = Kafka::Config.new
+        kafka = Kafka::Consumer.new(config)
+        kafka.should be_a(Kafka::Consumer)
```

Comment:
NOTE: This really isn't much of a test, just wanted something which would show the failure